db/migrate/20150107050243_add_index_to_votes.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexToVotes < ActiveRecord::Migration
+  def change
+    add_index :votes, [:user_id, :review_id], unique: true
+    add_index :votes, :review_id
```

Comment:
it's also likely that you want to index the `:user_id` - there is a use case for seeing a given user's votes.